src/components/__tests__/ResultTable.test.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import ResultTable from '../ResultTable'
+import { FormContext } from '@/contexts/context'
+import { initialFormState } from '@/defaults/defaultFormState'
+
```

Comment:
I ResultTableComponent se gjerne å bruke useMemo for operasjonene som er litt tyngre. Jeg tenker på disse array.find f.eks.. Tenk på at det koden som står i komponenten vil bli kjørt for hver render og vil koste mye i ytelse.